src/rest.js

```diff
@@ -82,7 +82,7 @@ export class Rest {
     return this.client.fetch(path, requestOptions).then((response: Response) => {
       if (response.status >= 200 && response.status < 400) {
         if (responseOutput) {
-          responseOutput.response = response;
+          responseOutput.response = response.clone();
```

Comment:
this line is the fix. The response needs to be cloned since the original will get consumed 2 lines bellow with the response.json(). 

The whole thing wasn't the best solution in the first place. aurelia-api only was made with json in mind. that responseObject is quite hacky. I'd recommend to get that fixed at least and then considering if it should be replaced by an better support for non json responses